mne_bids_pipeline/_config.py

```diff
@@ -202,7 +213,7 @@
 covariance (via `noise_cov='rest'`).
 """
 
-ch_types: Iterable[Literal["meg", "mag", "grad", "eeg"]] = []
+ch_types: Annotated[Sequence[Literal["meg", "mag", "grad", "eeg"]], Len(1, 4)] = []
```

Comment:
I errantly had `ch_types = "meg"`. By changing this from `Iterable` to `Sequence`, I got a nice error message saying it couldn't just be a string. Then we can use `pydantic` to further validate the length such as non-emptiness using `Len` from `annotated_types`, which is required by `pydantic`. So the deletions in `_config_import.py` below are now handled by pydantic for us!